app/assets/stylesheets/sass/buttons.scss

```diff
@@ -23,6 +23,7 @@ input.review-crud-buttons {
   @include buttonstyles;
   &:hover{
     @include buttonhovers(yellow);
+    color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.